hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -276,6 +282,24 @@ def _get_3par_vol_name(self, volume_id):
         volume_name = self._encode_name(volume_id)
         return "dcv-%s" % volume_name
 
+    def _get_3par_snap_name(self, snapshot_id):
+        """Get converted 3PAR volume name.
```

Comment:
chage description to 'get converted 3PAR snapshot name'

Reply:
Okay